dataactbroker/handlers/accountHandler.py

```diff
@@ -582,11 +583,11 @@ def update_user(self, system_email):
         if 'is_active' in request_dict:
             is_active = bool(request_dict['is_active'])
             if not user.is_active and is_active:
-                # Reset password count to 0
-                self.reset_password_count(user)
+                if is_local:
+                    # Reset password count to 0
+                    self.reset_password_count(user)
```

Comment:
Yeah let's get rid of it. I already removed the reset_password_email call. If you're running locally you'll be using the admin account from the config and we don't wanna accidentally log that out. Should we also get rid of the 120 days check to see if a user should be marked inactive?